### PR TITLE
Added a known issue in Usage for the error

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
`409 Conflict` response:
{"type":"UniqueConstraintViolationException","message":"Failed to create the record. The system administrator has define (truncated...)

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -263,4 +268,34 @@ Following section explains how to get a list of closed change requests (CRs) sin
 
 2. Click **Apply Latest Content** tile under the **Production** tab.
 
-3. Click **Fetch Latest Changes** button from the lower-left of the screen to get the list of issues resolved since the last applied change.
\ No newline at end of file
+3. Click **Fetch Latest Changes** button from the lower-left of the screen to get the list of issues resolved since the last applied change.
+
+## Known Issues
+
+Renaming a record, after raising the PR, creates a duplicate record with the same UUID when the changes are pushed through the PR. This action results in the import process (*Apply Latest Content*) failing.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Suggestion: Renaming a record in FortiSOAR, after raising the PR, creates a duplicate file/folder with the same UUID results in the import process......

File: docs/usage.md [hunk 2]
```diff
@@ -263,4 +268,34 @@ Following section explains how to get a list of closed change requests (CRs) sin
...
+
+```
+409 Conflict response:
+{"type":"UniqueConstraintViolationException","message":"Failed to create the record.
+The system administrator has define (truncated...)
+```
+
+### Workaround
+
+After the records have been renamed and before pushing the changes for review, delete the old records. Refer to [Deleting files in a repository - GitHub](https://docs.github.com/en/repositories/working-with-files/managing-files/deleting-files-in-a-repository) for deleting the file from GitHub.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Suggestion: After the records have been renamed and before pushing the changes for review, delete the old file/folder from the PR. Refer to.....

File: docs/usage.md [hunk 3]
```diff
@@ -25,6 +25,11 @@ The following pointers help understand the various steps of this lifecycle:
 
 5. Content developer selects the CR again and clicks **Submit Changes for Review**.
 
+    <table>
+        <th>NOTE</th>
+        <td>It is strongly recommended not to rename the records <em>after</em> the changes have been submitted for review. However, if you have to rename, refer to the section <a href="#known-issues">Known Issues</a>.</td>
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: We should not call this as a "known-issues". Can we call it as Recommended Step or Steps to push renamed file/folder.

Comment (cs-abhishek-shukla) [reply docs/usage.md]: Changed

File: docs/usage.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Changed

File: docs/usage.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Changed. Did not add "FortiSOAR" as these actions are specific to FortiSOAR application.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**